### PR TITLE
Change text size for getting started page 2656

### LINKED ISSUE #2656: Changing text size for Getting Started page
### Dependency
- [x] ~https://github.com/hackforla/website/issues/2690~

### Overview
We need to make sure the text sizes for the header section on the Getting Started page is consistent with the rest of the site.

### Action Items
- [x] Review the text sizes of all the project pages.
- [x] Change the text size of Getting Started page by using the SCSS class `paragraph2` for the paragraphs in the header section so it is consistent with other project pages. (See the Paragraphs section under Typography in the [Design System in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3689%3A21144) for how paragraph classes are defined.)
- [x] This issue should also address cleaning up the code in the .header-p class (where it would overlap with the properties in the paragraph2 class). Here is a link to the .header-p class: https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/_sass/components/_getting-started-page.scss#L16-L32
- [x] Make sure the page looks fine for smaller viewport screens as well, such as tablet, mobile (responsiveness)

### Resources/Instructions
- See the Paragraphs section under Typography in the [Design System in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3689%3A21144) for how paragraph classes are defined
- See the [Getting Started page design in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=11987%3A91947) in the red box
- Link to [Getting Started page](https://www.hackforla.org/getting-started)
- Link to [Getting Started page SCSS code](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_getting-started-page.scss)
- Link to [Getting Started page HTML code](https://github.com/hackforla/website/blob/8b0d5bffa7c6757340f673e1dad6d10ecd6bc5ec/pages/getting-started.html)

#### Screenshot of the paragraph text size to be changed as per Figma
Getting started page paragraph font size

### PR DESCRIPTION
**Pull request by A-BMT02**
Fixes #2656 
Changed text size for Getting Started Page

  - Added "paragraph2" class to Getting Started page as requested
  - Removed 4 lines of code form "header-p" class as it overlaps with the new "paragraph2" class added
  
**"Getting started" page before change** 
Screenshot (158)

**"Getting Started" page after change**
Screenshot (160)

### CODE REVIEW

**Comment by SAUMILDHANKAR** [comment]
@A-BMT02 Just want to share the following points with you regarding the review process and some other processes that we are currently following in our team:
- I have removed the reviewers from this PR as currently we assign reviewers during the team meetings. Requesting to not assign reviewers yourself since they may not be available or part of our team and doing so may also prevent other team members from picking up this PR for review.
- Also, requesting you to review some of the PRs as well while you are waiting for yours to be reviewed.
- Please add Fixes #issuenumber in your top comment as well as a note about if before and after images are required or not. Adding following two links for more information: [Contributing guideline for adding issue number to your PR](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#ii-complete-pull-request-2-add-issue-number-to-the-pull-request) and [note about adding images](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#iv-complete-pull-request-4-include-images-if-available)
- We are now requesting all website team members to attend at least one meeting in a week (Tuesday, Thursday or Sunday) as we strive to replicate a professional working environment within our team amongst other things. Therefore, request you to let us know which of the weekly meetings would be suitable for you based on availability. We have decided to make an exception for this requirement only on a case-by-case basis.

Please feel free to reach out to me through slack or leave a comment and @ mention my GitHub username for more information. Thank you.

**Comment by SAUMILDHANKAR** [comment]
Review ETA: End of day 3/21/22
Availability: Monday 5-6 PM Pacific Time

**Comment by frankstepanski** [comment]
Review ETA: End of day 3/21/22
Availability: Monday evening EST

**Comment by A-BMT02** [comment]
@SAUMILDHANKAR Thank you for the update 
Noted and appreciated

As for the meetings , i will be attending the ones on Sundays

**Comment by A-BMT02** [comment]
@SAUMILDHANKAR Also,  what is the process for reviewing a pull request ? 

Do i have to be assigned a pull request to review IN A MEETING?

**Review by frankstepanski — DISMISSED** [review]
Great job @A-BMT02 adding the paragraph2 class and removing the header-p CSS class.
Tested the layout and displays as expected.

**Comment by tamara-snyder** [comment]
@A-BMT02 You may self-assign a pull request at any time, not just at the meetings. Here's a link to Hack for LA's guide on [How to Review Pull Requests](https://github.com/hackforla/website/wiki/How-to-Review-Pull-Requests), which should tell you all you need to know.

If you have any questions, feel free to drop a message in the [hfla-site-pr channel](https://hackforla.slack.com/archives/C025ERFDM4Y) on Slack.

**Comment by A-BMT02** [comment]
Thank you @tamara-snyder

File: _sass/components/_getting-started-page.scss [hunk 1]
```diff
@@ -14,15 +14,12 @@
...
   display: flex;
   max-width: 580px;
   text-align: center;
   padding: 0 30px 10px 30px;
   // padding-bottom: 10px;
   margin: 12px auto;
-  font-family: "Roboto"sans-serif;
 
   @media #{$bp-below-tablet} {
     font-size: 18px;
```
Comment (SAUMILDHANKAR) [inline _sass/components/_getting-started-page.scss:25]: Please consider if this font-size would interfere with paragraph2 class in mobile view.

Comment (SAUMILDHANKAR) [inline _sass/components/_getting-started-page.scss]: Please consider if this font size would interfere with paragraph2 class in mobile view.

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
@A-BMT02 Great work in adding the paragraph2 class and making changes to .header-p class. Requesting to please add Fixes #issuenumber in your top comment as well as a note about if before and after images are required or not. Refer to the following two links for more information: [Contributing guideline for adding issue number to your PR](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#ii-complete-pull-request-2-add-issue-number-to-the-pull-request) and [note about adding images](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#iv-complete-pull-request-4-include-images-if-available)

File: _sass/components/_getting-started-page.scss (hunk 1 above)
Comment (A-BMT02) [reply _sass/components/_getting-started-page.scss]: @SAUMILDHANKAR You are right , the font-size in the media is redundant 
I removed it

**Comment by A-BMT02** [comment]
@SAUMILDHANKAR Thank you for the review
Made the necessary changes

**Comment by SAUMILDHANKAR** [comment]
Re-review ETA: End of day 3/24/22
Availability: Thursday 4-5 PM Pacific Time

**Comment by SAUMILDHANKAR** [comment]
Revised Re-review ETA: End of day 3/25/22
Availability: Friday 7-8 PM Pacific Time
@A-BMT02 I was not able to finish my review yesterday, but I plan to do it today. Sorry for the delay. Thak you.

**Comment by A-BMT02** [comment]
@SAUMILDHANKAR No problem

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
@A-BMT02 Thanks a lot for removing the font-size for mobile view. Request you to also change your initial comment about "removed 3 lines of code from header-p" to "removed 4 lines of code from header-p" and include the before and after images as well. I understand that changes are hardly noticeable, but it would be easier for QA and would be consistent with the process that we follow. Really appreciate your good work on this issue. Once you have made the changes, please request a re-review by clicking on the icon next to my GitHub handle on top. Thank you.

**Comment by A-BMT02** [comment]
@SAUMILDHANKAR 
Updated my initial comment
Thanks for the review , i'm loving it!

**Comment by kathrynsilvaconway** [comment]
Review ETA: 10AM EST on 3/29/2022
Availability: Tuesday Morning EST

**Comment by kathrynsilvaconway** [comment]
Update on availability:

Review ETA: 10PM EST on 3/29/2022
Availability: Tuesday Afternoon/Evening EST

I am still working out my PR review skills. It took me a little longer than I expected to view all the changes. I will write my review and have it up before the meeting tonight.

**Comment by A-BMT02** [comment]
@kathrynsilvaconway Thats totally ok , i'm also working on my first pull request review too
Take your time

**Review by SAUMILDHANKAR — APPROVED** [review]
@A-BMT02 Thanks for making the changes. Looks good in my environment. I will wait for Kathryn's review. Good job!

**Review by kathrynsilvaconway — APPROVED** [review]
Nice work adjusting the paragraph on the Getting Started page, @A-BMT02. I see that the 'paragraph2' tag has been added and that you have removed the lines from the .header-p class that were redundant. The page looks good in all the mobile checks as well.

**Comment by A-BMT02** [comment]
Thank you @kathrynsilvaconway

**Pull request merged**